src/services/quota/index.tsx

```diff
@@ -16,6 +16,13 @@ export class NotEligibleError extends Error {
   }
 }
 
+export class AuthenticationError extends Error {
```

Comment:
How is this different from the set of `Auth.*Error` in the auth service?

Reply:
this is not exactly a `LoginError`, so i didnt want to reuse the errors in the auth service. i named it differently to avoid confusion 😅

Reply:
It's challenging to differentiate between `AuthenticationError` and `AuthInvalidError`/`AuthExpiredError`. Can we name this better to avoid confusion?

Reply:
how about `SessionError`?

Reply:
Sorry just saw this, Yep sounds good! Are there other types of session errors? Or only expiry will make sense?

Reply:
np! we were mainly testing it using expiry (we manually change the expiry date in the db after logging in). but that shouldn't happen realistically because we have the `useValidateExpiry` hook and the backend returns a ttl on login. 
interestingly, our backend won't check for the role of the user in the entire login flow. so creating an user with role that is not "OPERATOR" would trigger this type of error also for any requests that go thru `onlyAuthorizedOperatorSession` in the backend

Reply:
I see, thanks for explaining! Ok sounds good then, for now we just stick to a generic SessionError, in future we may include other types.